featuretools/computational_backends/calculate_feature_matrix.py

```diff
@@ -110,6 +113,11 @@ def calculate_feature_matrix(features, cutoff_time=None, instance_ids=None,
     if not isinstance(cutoff_time, pd.DataFrame):
         if cutoff_time is None:
             cutoff_time = datetime.now()
+            # if integer time index, use max value as cutoff time instead
+            if target_entity.time_index:
```

Comment:
the target entity might not have a time index, but some other entity in the entityset might. perhaps it makes sense to add a method or attribute to an entityset that returns if it is a datetime or numeric time index. 

we should assume all entities in an entityset are using the same units for the time index, so this method could do the check when being initialized or when `set_time_index` is called.